readme.md

```diff
@@ -269,6 +269,20 @@ The following properties are defined:
 * `endpoint`: Base URL for the API, without e.g. the `index/graphql` suffix.
 * `apiVersion`: One of `otp1`, `otp2` or `entur`.
 
+#### TRIAS
+
+*WIP*
+
+The following properties are defined:
+* `endpoint`: Base URL for the API
+* `testingEndpoint`: Base URL for testing purposes
+* `apiVersion`: `1.X` as [available](https://github.com/VDVde/TRIAS/releases)
+* `content-type`: [Content-Type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type) e.g. `text/xml`
+* `openAPI`: Boolean, whether one need to sign up or not
+* `authorizationMethod`: One of `Authorization-Header`
+* `providedServices`: array of supported services as listed [in the documentation](https://www.vdv.de/431-2sdes-v-1-3.pdfx#page=51)
+* `requestLimit`: requests per day, which are allowed
```

Comment:
Shall we follow the [markup of the `RateLimit` HTTP header](https://www.ietf.org/archive/id/draft-ietf-httpapi-ratelimit-headers-04.html#name-expressing-rate-limit-polic) here? It is an established way of specifying the rate limit(s) of an API.

Reply:
This looks good, from a first view. Thanks! I will look into it and update the documentation accordingly.